Added  requested version for csstype

saraswati-patra opened the pull request:
## Checklist
<!--- Goto Preview tab for better readability -->
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

seth-priya commented:
@saraswati-patra  is the inclusion of the "csstype_RHEL7.6.sh.bak" file in this commit intentional?

saraswati-patra commented:
> @saraswati-patra is the inclusion of the "csstype_RHEL7.6.sh.bak" file in this commit intentional?

Added .bak file wrongly,Deleted and commit.thanks

seth-priya commented on c/csstype/csstype_RHEL7.6.sh:
```diff
@@ -2,11 +2,13 @@
-# Maintainer    : Vedang Wartikar <vedang.wartikar@ibm.com>
+# Version       : v2.6.0,v2.6.9
+# Source repo   : https://github.com/frenic/csstype
+# Tested on     : RHEL 7.6,8.5
```
this should be RHEL 7.6, RHEL 8.5 for consistency

seth-priya commented:
@vathsalashetty25 @kumarvikas2020 if @saraswati-patra  can't , can one of you handle this?
